### ISSUE TITLE
1.17.1: /ignore command clashes with GriefPrevention's /ignore, no way to specify priority in commands.yml

### ISSUE THREAD

**Issue by Jupitour** [kangarko/ChatControl#1228]
Hi there, when running the **/ignore** command, GriefPrevention seems to take precedent and run ignore, but it doesn't actually ignore players in chat. I'm not sure how to increase the priority of ChatControl's ignore command, or even specify it to be run in **commands.yml** since there's no /chc:ignore

TL;DR, ignore doesn't work for me with GriefPrevention installed, and both GriefPrevention and ChatControlRed are indispensible for me. Could there be a possible fix?

Minecraft Version: 1.17.1
MySQL: false
BungeeCord: false

**Comment by ElBananaa** [comment]
> Hi there, when running the **/ignore** command, GriefPrevention seems to take precedent and run ignore, but it doesn't actually ignore players in chat. I'm not sure how to increase the priority of ChatControl's ignore command, or even specify it to be run in **commands.yml** since there's no /chc:ignore
> 
> TL;DR, ignore doesn't work for me with GriefPrevention installed, and both GriefPrevention and ChatControlRed are indispensible for me. Could there be a possible fix?
> 
> Minecraft Version: 1.17.1
> MySQL: false
> BungeeCord: false

Hey!
Please tey to play around with the listerner priority in chatcontrol's settings file. Try to increase it (don't forget to restart the server each time you change the listener priority), and see if it changes anything.

**Comment by Jupitour** [comment]
Will do, I'll update this issue on the results.

**Comment by Jupitour** [comment]
Hi there, I set it to "HIGHEST" but it still doesn't work. I'm a bit wary of setting it to MONITOR though?

Also, yes I did restart the server when the listener is changed

**Comment by ElBananaa** [comment]
> Hi there, I set it to "HIGHEST" but it still doesn't work. I'm a bit wary of setting it to MONITOR though?
> 
> Also, yes I did restart the server when the listener is changed

Hey,
You shouldn't be too scared of playing with the listener priority, setting it to MONITOR may totally fix your issue, without causing any others. I'd suggest at least trying with it, and see if it fixes this issues/causes other issues ^^

**Comment by Jupitour** [comment]
I'll give it a go then :) thank you for the reply!

**Comment by Jupitour** [comment]
Hi there, setting it to MONITOR didn't do anything. When running **/ignore** GriefPrevention still picks up the command instead of CHC Red

**Comment by ElBananaa** [comment]
Hey!
Thanks for giving this a shot!
I also tried it on my end, and I wasn't able to reproduce your issue, even with the default Listener priority in both plugins.
Are you sure that you enabled our ignore command in the settings file? (You must set `Ignore.Enable` to `true`)?
Would you mind providing us your `/chc debug` output file so I can try to reproduce your issue with your current ChatControlRed setup?


Anyways, in case the command is already correctly enabled, in order to fix this issue, you can simply override GriefPrevention ignore command by using the `commands.yml` file in your server's root directory.
Put this inside of you file, and it should fix your issue:
```yaml
aliases:
  icanhasbukkit:
  - version $1-
  ignore:
    - ig $1-
```

Let me know if it works and if you need anything else! :)

**Comment by kangarko** [comment]
If you want to use ChatControl's ignore system, enable it in our Ignore section of settings.yml.

If you want to force our ignore system over GriefPrevention, you need to open GriefPrevention JAR file with WinRar or similar, then open its plugin.yml and remove the whole ignore section from commands section manually each time you update that plugin.
